src/datasources/push-notifications-api/firebase-cloud-messaging-api.service.ts

```diff
@@ -0,0 +1,141 @@
+import { IConfigurationService } from '@/config/configuration.service.interface';
+import { CacheRouter } from '@/datasources/cache/cache.router';
+import {
+  CacheService,
+  ICacheService,
+} from '@/datasources/cache/cache.service.interface';
+import { HttpErrorFactory } from '@/datasources/errors/http-error-factory';
+import {
+  NetworkService,
+  INetworkService,
+} from '@/datasources/network/network.service.interface';
+import { IPushNotificationsApi } from '@/domain/interfaces/push-notifications-api.interface';
+import { Inject, Injectable } from '@nestjs/common';
+import { FirebaseNotification } from '@/datasources/push-notifications-api/entities/firebase-notification.entity';
+
+// TODO: Refactor to use JwtService
+import jwt from 'jsonwebtoken';
```

Comment:
We currently don't expose the option to sign a JWT with a specific secret as a certain issue. I will create a follow up PR that refactors the implementation and uses it here instead of this package directly.

Reply:
I've opened [a PR with the refactor](https://github.com/safe-global/safe-client-gateway/pull/1768), where I'll branch from to add the aforementioned tests.